ftw/iframeblock/browser/resources/main.js

```diff
@@ -1 +1 @@
-require(['iframeblock', 'iframeResizer'], function (iframeblock, iframeResizer) {});
+require(['iframeblock', 'iframeResizer', 'iframeResizer.contentWindow'], function (iframeblock, iframeResizer, iframeResizer_contentWindow) {});
```

Comment:
As the quote above says it is needed to be able to use it. There are some cases where a customer wants to create an iframe from a form on its website on some content page. Then we need the `contentWindow` js otherwise it won't resize the form iframe: 

Example `ContentPage` with a iFrame to the `Plone Site`
---
without contentWindow js
---
![image](https://user-images.githubusercontent.com/9467802/76067085-15449680-5f8f-11ea-91e4-509fac70f55c.png)

with contentWindow js
---
![image](https://user-images.githubusercontent.com/9467802/76067122-24c3df80-5f8f-11ea-971c-0b66d87112f4.png)

Reply:
Ok, I guess... Are they doing that already?

Anyone done an iframe feedback loop yet? ;-). https://www.youtube.com/watch?v=nb1OMNlPdBc